### ISSUE TITLE
[Bug]: Workflow UI is broken on 25

### ISSUE THREAD

**Issue by come-nc** [nextcloud/server#34395]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When using 25, the Flow settings UI is not correctly working:

- The selected events do not appear as selected in the multiselect, the checkmark is not showing
- Some flows do not allow selecting an event, it selects one and clicking do nothing
- The arrow has a wide left margin, resulting on the right part wrapping on the next line with a window a bit less large than fullscreen

### Steps to reproduce

1. Install and activate several flow apps
2. Go into admin settings
3. Add and configure flows

### Expected behavior

Same UI as 24 or better

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
Default ones + flows
```

**Comment by come-nc** [comment]
@eneiluj @blizzz ^

**Comment by come-nc** [comment]
> * Some flows do not allow selecting an event, it selects one and clicking do nothing

This one is actually on purpose, some flows are only able to act on file changed. The french translation makes it confusing which is why I thought this was not expected.

**Comment by come-nc** [comment]
Also, when selecting mime type, second select defaults to «matches» which should only be proposed for regular exceptions, and gives a confusing error when validating, see https://github.com/MCMic/workflow_kitinerary/issues/32#issue-1524479284

**Comment by blizzz** [comment]
maybe someone from @nextcloud/server-frontend  can give it a hand?

**Comment by Pytal** [comment]
> maybe someone from @nextcloud/server-frontend can give it a hand?

Yup :) tracking in https://github.com/nextcloud/server/issues/36125

**Comment by szaimen** [comment]
I think this is resolved in the meantime on nc27?

**Comment by joshtrichards** [comment]
#38170

**Comment by blizzz** [comment]
With #38170 being merged, is here something left to do?

**Comment by come-nc** [comment]
Could still be improved, but yes, it’s better and it’s working, closing.
